code/mod_table/mod_table.cpp

```diff
@@ -1136,6 +1137,10 @@ void parse_mod_table(const char *filename)
 			stuff_boolean(&Hotkey_always_hide_hidden_ships);
 		}
 
+		if (optional_string("$Allow automatically assigning personas:")) {
+			stuff_boolean(&Auto_assign_personas);
```

Comment:
Almost every boolean flag in game_settings or ai_profiles is something that defaults to false and is optionally set to true.  This should be consistent with that, e.g. `$Do not automatically assign personas:`

Reply:
When you go adjust `$AI use framerate independent turning:` and the few ai_profiles flags I found to make this a true statement across the board then I'll bother with this. Till then, don't merge it, I guess. I don't have time for petty OCD stuff.

Reply:
The fact that a standard has not been met in 100% of cases is not a valid reason to abandon the standard.

Reply:
That's an interesting way of looking at it.  But the original pattern was not "do something that isn't normally done" vs "stop doing something that's normally done", but rather "does this alter the base game behavior".  That produces an easy way to check flags, both in mods and in the code: if it alters the behavior, it's true; if it doesn't alter it, it's false.

Reply:
Any other coders feel like weighing in?

Reply:
Hmm I can see both, if we are favoring modders though having the logic match the flag would be the most useful for modders IMO.